docs/sources/developers/plugins/cross-plugin-linking.md

```diff
@@ -1,21 +1,23 @@
 ---
-title: Cross plugin links
+title: Work with cross-plugin links
```

Comment:
Potentially a follow up, but I think we need to consider the title some more, its not clear this is to do with extensions.

Reply:
Do you have an alternative suggestion? If not, let's revisit this in phase 3.